src/OOP.java

```diff
@@ -4,5 +4,10 @@ public static void main(String[] args) {
         System.out.println(pot.high);
         System.out.println(pot.width);
         System.out.println(pot.color);
+        User user = new User (7,"Adam");
+        if (user.ifcanbuyalcohol()){
```

Comment:
method name should be written in camelCase notation like `ifCanBuyAlcohol`

Reply:
Done